Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use a minimal base image with NVIDIA GPU support
+FROM livepeer/go-livepeer:0.7.1
```

Comment:
@Titan-Node, I considered using a more lightweight image for a smaller footprint: [nvidia/cuda:11.8.0-base-ubuntu22.04](https://hub.docker.com/layers/nvidia/cuda/11.8.0-base-ubuntu22.04/images/sha256-dd091e83937b98bd288843d1a5401e11798b91661abf83857e20a6ebbec03100?context=explore)

```dockerfile
FROM nvidia/cuda:11.8.0-base-ubuntu22.04
```

While attempting to streamline the image, however, I encountered a hurdle. I have yet to pinpoint the minimal dependencies required from the [Dockerfile](https://github.com/livepeer/go-livepeer/blob/master/docker/Dockerfile) to ensure smooth operation. Upon initiating the pool, an issue surfaced with the message `Error while parsing '-nvidia all' flag: no devices found with vendor name 'Nvidia'`. I assume that you might find it more accessible to troubleshoot, given your access to the binary code. Your insights could pave the way for a seamless resolution.

Reply:
@Titan-Node I ran more tests, and as discussed in our Discord conversation, the smaller container faces challenges only when the `-nvidia all` flag is used or when users omit the `--gpus IDs` Docker argument (which defaults to `-nvidia all`):

> Hey @titannode, @icsy7867. I've quickly examined the v6.0.x go-livepeer client in the base image and identified a challenge related to GPU detection within a Docker container during benchmarking. The root cause seems to be the dependency on the https://pkg.go.dev/github.com/jaypipes/ghw@v0.10.0 package, which, in turn, relies on https://pkg.go.dev/io/ioutil for GPU detection.
>
>Specifically, when using the `-nvidia all` option, the go-livepeer client parses the vendor information of connected devices, querying data from the `/sys/class/drm/` folder via ioutil. However, the information in this folder may be slightly modified in the container environment, leading to unsuccessful GPU detection.
>
>Interestingly, the issue is not replicated when using the FROM livepeer/go-livepeer:0.7.1 base. I suspect that there might be a discrepancy in device information mapping within the base Docker container, a difference resolved in the go-livepeer Dockerfile.

>If you have any insights into resolving this matter, please share your thoughts. Otherwise, for users intending to run the pool in a Docker container, the current workaround involves using the larger `livepeer/go-livepeer` Docker base or specifying GPU numbers explicitly.

For resolution, two paths emerge:

1. Embrace the larger `livepeer/go-livepeer` Docker base.
2. Opt for the smaller container and update the documentation, cautioning users to explicitly specify their GPU IDs when running the pool in a Docker environment.

@Titan-Node, kindly inform me of your preference, and I'll adjust the PR accordingly 👍🏻.